gui/re.chiaki.Chiaki.appdata.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!-- Copyright 2020 Florian Märkl <chiaki@metallic.software> -->
 <component type="desktop">
-  <id>re.chiaki.Chiaki</id>
+  <id>re.chiaki.chiaki</id>
```

Comment:
Why? Uppercase in the app name part of id is not a problem at all and is commonly used. For example, most GNOME apps including GNOME Software app itself have its app name in uppercase.

Also, as far as I know, the id **should** definitely match the appdata file name. otherwise it would cause issues for most AppStream parsers.

Reply:
https://www.freedesktop.org/software/appstream/docs/chap-Metadata.html#spec-component-filespec
<quote>it is strongly encouraged to only use lowercase letters for the ID</quote>

 (fenio@debian) ➜  metainfo appstreamcli validate re.chiaki.Chiaki.appdata.xml 
W: re.chiaki.Chiaki:16: description-has-plaintext-url p
I: re.chiaki.Chiaki:4: cid-contains-uppercase-letter re.chiaki.Chiaki

Validation failed: warnings: 1, infos: 1, pedantic: 1

Are you saying that appstream validator is broken wrt to that?

Reply:
```
$ cat /usr/share/metainfo/org.gnome.Software.appdata.xml|head
<?xml version="1.0" encoding="UTF-8"?>
<!-- Copyright 2013-2016 Richard Hughes <richard@hughsie.com> -->
<component type="desktop">
  <id>org.gnome.Software.desktop</id>
```

This is how the id of GNOME Software itself (and most other GNOME apps) looks like. :-) Anyway, as I said, feel free to change it, but please also change the file name.

Reply:
Anyway, just change the file name to match the id + also don't forget to change the name in gui/CMakeLists.txt and it should be fine.